### PR TITLE
Added support for upload files using uploadables option

### PR DESCRIPTION
**Pull request by hariroshan**
Can you check and merge?

### CODE REVIEW

File: packages/reason-relay/src/ReasonRelay.re [hunk 1]
```diff
@@ -431,6 +431,11 @@ module Network = {
   type t;
 
   type operation = {
+    /** This will be available only when using persisted queries. 
+     * And id field will be of presisted query Id. 
+     * In other cases, it will be null */
+    id: string,
+    /** When using persisted queries, text field will be null. */
```
Comment (zth) [inline packages/reason-relay/src/ReasonRelay.re:438]: You can go ahead and remove this comment in the implementation file. It's already in the `rei` file and that should be enough.

**Review by zth — CHANGES_REQUESTED** [review]
Hi, thanks for this! I think this diff is slightly out of date - would you mind rebasing to master, and also remove the package lock file (we prefer `yarn`)? I'll be happy to take a look/merge after.

File: packages/reason-relay/src/ReasonRelay.re (hunk 1 above)
Comment (hariroshan) [reply packages/reason-relay/src/ReasonRelay.re]: I moved the comments to rei file. I don't get the rebase part. The command says I'm already up-to date

Comment (zth) [reply packages/reason-relay/src/ReasonRelay.re]: Thanks! Yeah that's weird, there still seems to be conflicts to resolve. Maybe try and resolve them in the web UI?

Comment (hariroshan) [reply packages/reason-relay/src/ReasonRelay.re]: I think it worked now.

**Review by zth — APPROVED** [review]
Excellent! The nullability is the only thing I had in mind, but I saw you fixed that in a recent commit too 👍 I'll merge this and try getting a new release within the coming days. I just have another thing I'd like to finish up first. 

Thank you for your contribution!

**Pull request merged**